docs/manual/common/gettingstarted/LagomExamples.md

```diff
@@ -3,19 +3,17 @@
 After getting started with Lagom's Hello World example, you can learn more about Lagom by downloading and
 running Lagom's "Shopping Cart" example (either [shopping-cart-scala][] or [shopping-cart-java][]).
 
-[shopping-cart-scala]: https://github.com/lagom/lagom-samples/tree/1.5.x/shopping-cart/shopping-cart-scala
-[shopping-cart-java]: https://github.com/lagom/lagom-samples/tree/1.5.x/shopping-cart/shopping-cart-java
+[shopping-cart-scala]: https://github.com/lagom/lagom-samples/tree/1.6.x/shopping-cart/shopping-cart-scala
+[shopping-cart-java]: https://github.com/lagom/lagom-samples/tree/1.6.x/shopping-cart/shopping-cart-java
 
 Some of the Lagom features you can observe include:
 
-Feature |Chirper| Online Auction |
---------|:--------:|:-------------:|
-How to use persistence| Y | Y 
-How to use the message broker API| N | Y
-How to communicate between services| Y | Y
-How streaming service calls work | Y | N
-How to develop a Play app with Lagom | Y | Y
-How to consume Lagom services from a front-end Play app | N | Y
+* How to use persistence
+* How to use the message broker API
+* How to communicate between services
+* How streaming service calls work
+* How to develop a Play app with Lagom
+* How to consume Lagom services from a front-end Play app
```

Comment:
Shopping cart doesn't showcase the last three items on the list.

Chirper demo-ed websockets (aka, streaming calls) and the interaction with Play is not demoed at all.

Reply:
Actually, without chirper and online auction I don't think there's any sample demoing integration nor streaming calls.